aspnetcore/blazor/supported-platforms.md

```diff
@@ -0,0 +1,27 @@
+---
+title: ASP.NET Core Blazor supported platforms
+author: guardrex
+description: Learn about the supported platforms for ASP.NET Core Blazor.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 05/21/2019
+uid: blazor/supported-platforms
+---
+# ASP.NET Core Blazor supported platforms
+
+By [Luke Latham](https://github.com/guardrex)
+
+## Server system requirements
```

Comment:
I know this section was copied from the SignalR supported platforms doc, but I think it's self-evident and can be removed. Blazor is part of ASP.NET Core, so it should go without saying that it works wherever ASP.NET Core works.